spec_creation/autofill_eval_spec.py

```diff
@@ -156,9 +156,32 @@ def get_route_from_relation(r):
     return get_coords_for_relation(r["relation_id"],
         r["start_node"], r["end_node"])
 
-def validate_and_fill_leg(orig_leg):
-    # print(t)
+def _add_temporal_ground_truth(orig_loc, start_fmt_date, end_fmt_date):
```

Comment:
I would rename to clarify that these are the default start and end dates since we only use them if the loc entry doesn't have any values.